image-layout.md

```diff
@@ -7,7 +7,7 @@ Given an image layout a tool can convert a given ref into a runnable OCI Image F
 The image layout has two top level directories:
 
 - "blobs" contains content-addressable blobs. A blob has no schema and should be considered opaque.
-- "refs" contains descriptors pointing to an image manifest list
+- "refs" contains descriptors pointing to an image manifest
```

Comment:
I don't get this, referenced blobs may be missing from `./blob/` but it's a thing that `./refs` contains  manifest (either single or list)